force-app/main/default/lwc/githubViewerCommitDetails/githubViewerCommitDetails.css

```diff
@@ -0,0 +1,31 @@
+.commit-container {
```

Comment:
[INFO]
Nie jestem CSSowym świrkiem, więc jeśli działa to git xD ale ciężko mi dać Ci jakiś super feedback